doc/source/dev/howto-docs.rst

```diff
@@ -0,0 +1,109 @@
+.. _howto-docs:
+
+############################################
+How to contribute to the NumPy documentation
+############################################
+
+The *Documentation* for a software project is the set of reference,
+instructional, educational, informative material generated by the project
+developers and contributors, as well as discussions, presentations, videos and
+other user generated content. It may include learning-oriented content (such as
+Tutorials and How Tos), Use Cases or in depth explanations and reference for
+developers. 
+
+If you're reading this page, you probably want to help. This guide is meant to
+help you decide which kind of content you'll write, as well as give you some
+tips and instructions for submitting it to the official NumPy documentation
+(that is, the documentation that ships with NumPy and lives on the official
+project pages). Keep in mind that if you don't want to do this, writing a
+tutorial on your own blog, creating a youtube video or answering questions on
+social media or the slack channel are also great contributions! 
+
+NumPy has a Documentation team. We have open meetings on Zoom every three weeks
+and invite everyone to join. Don't hesitate to reach out if you have questions
+or just need someone to guide you through your first steps - we're always happy
+to help. Meetings are usually announced on the `numpy-discussion mailing list
+<https://mail.python.org/mailman/listinfo/numpy-discussion>`_. Meeting minutes
+are stored `on hackmd.io <https://hackmd.io/oB_boakvRqKR-_2jRV-Qjg>`_.
+
+You can find a summary of our long term goals for the documentation `at our
+github project <https://github.com/orgs/numpy/projects/2>`_.
+
+Current vision for the documentation: NEP 44
+--------------------------------------------
+
+Recently, the NumPy community approved a *NumPy Enhancement Proposal (NEP)*
+about documentation. You can read it `here
+<https://www.numpy.org/neps/nep-0044-restructuring-numpy-docs>`_.
+
+**Where is the documentation?**
+
+The main page for the :ref:`NumPy Documentation <manual>` lists several
+categories. The documents mentioned there live in different places.
+
+- **Tutorials, How Tos, Explanations:** These documents are stored in the NumPy
+  source code tree, which means that in order to add them to the official
+  documentation, you have to download the NumPy source code, build it and
```

Comment:
maybe needs a link to [``howto-build-docs``](https://numpy.org/devdocs/docs/howto_build_docs.html)